app-emulation/virtualbox-guest-additions/virtualbox-guest-additions-7.0.2.ebuild

```diff
@@ -6,55 +6,58 @@ EAPI=8
 inherit edo linux-mod systemd toolchain-funcs udev
 
 MY_PN="VirtualBox"
-MY_PV="${PV/beta/BETA}"
-MY_PV="${MY_PV/rc/RC}"
-MY_P="${MY_PN}-${MY_PV}"
-[[ "${PV}" == *a ]] && DIR_PV="$(ver_cut 1-3)"
+MY_P="${MY_PN}-${PV}"
 
 DESCRIPTION="VirtualBox kernel modules and user-space tools for Gentoo guests"
 HOMEPAGE="https://www.virtualbox.org/"
-SRC_URI="https://download.virtualbox.org/virtualbox/${DIR_PV:-${MY_PV}}/${MY_P}.tar.bz2
+SRC_URI="https://download.virtualbox.org/virtualbox/${PV}/${MY_P}.tar.bz2
 	https://gitweb.gentoo.org/proj/virtualbox-patches.git/snapshot/virtualbox-patches-6.1.36.tar.bz2"
 
-LICENSE="GPL-2"
+LICENSE="GPL-2+ GPL-3 LGPL-2.1 MIT"
 SLOT="0/$(ver_cut 1-2)"
-[[ "${PV}" == *_beta* ]] || [[ "${PV}" == *_rc* ]] || \
-KEYWORDS="amd64 x86"
+KEYWORDS="~amd64 ~x86"
 IUSE="X"
 
 # automount Error: VBoxServiceAutoMountWorker: Group "vboxsf" does not exist
+# TODO: find out what this is, remove comment if obsolete
+
 RDEPEND="
 	acct-group/vboxguest
 	acct-group/vboxsf
 	acct-user/vboxguest
-	X? ( x11-apps/xrandr
+	sys-apps/dbus
```

Comment:
We have some opinionated users who prefer to keep dbus optional. What's it used for?

Reply:
Not sure.  But I'll add a USE flag.